src/utils/solanaUtils.ts

```diff
@@ -447,10 +565,38 @@ export const getCollectables = async (
 ) => {
   const collectables = (await metaplex
     .nfts()
-    .findAllByOwner({ owner: pubKey })
-    .run()) as Metadata<JsonMetadata<string>>[]
+    .findAllByOwner({ owner: pubKey })) as Metadata<JsonMetadata<string>>[]
 
-  return collectables
+  // TODO: Remove this filter once the uri is fixed for HOTSPOTS
+  const filteredCollectables = collectables.filter(
+    (c) => c.symbol !== 'HOTSPOT',
+  )
+
+  return filteredCollectables
+}
+
+/**
+ * Returns the account's collectables
+ * @param pubKey public key of the account
+ * @param oldestCollectable starting point for the query
+ * @returns collectables
+ * TODO: Need to add pagination via oldest collectable param in collectables slice
```

Comment:
Should we add pagination now?

Reply:
Don't want this PR getting any bigger but thats in my list of things todo next once I am unblocked on the nft compression side of things